modules/installation-disk-partitioning.adoc

```diff
@@ -0,0 +1,124 @@
+by creating a machine config that is inserted during the `openshift-install`
+preparation phases of an {product-title} installation.
+
+.Procedure
+
+. Create a directory to hold the {product-title} installation files:
++
+[source,terminal]
+----
+$ mkdir $HOME/clusterconfig
+----
+
+. Run `openshift-install` to create a set of files in the `manifest` and
+`openshift` subdirectories. Answer the system questions as you are prompted:
++
+[source,terminal]
+----
+$ openshift-install create manifests --dir $HOME/clusterconfig
+? SSH Public Key ...
+$ ls $HOME/clusterconfig/openshift/
+99_kubeadmin-password-secret.yaml
+99_openshift-cluster-api_master-machines-0.yaml
+99_openshift-cluster-api_master-machines-1.yaml
+99_openshift-cluster-api_master-machines-2.yaml
+...
+----
+
+. Create a `MachineConfig` object and add it to a file in the `openshift` directory.
+For example, name the file `98-var-partition.yaml`, change the disk device name to the name of the storage device on the `worker` systems, and set the storage size as appropriate. This attaches storage to a separate `/var`
+directory.
+
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 98-var-partition
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+    storage:
+      disks:
+      - device: /dev/<device_name> <1>
+        partitions:
+        - sizeMiB: <partition_size>
+          startMiB: <partition_start_offset> <2>
+          label: var
+      filesystems:
+        - path: /var
+          device: /dev/disk/by-partlabel/var
+          format: xfs
+    systemd:
+      units:
+        - name: var.mount
+          enabled: true
+          contents: |
+            [Unit]
+            Before=local-fs.target
+            [Mount]
+            Where=/var
+            What=/dev/disk/by-partlabel/var
+            [Install]
+            WantedBy=local-fs.target
+----
++
+<1> The storage device name of the disk that you want to partition.
+<2> When adding a data partition to the boot disk, a minimum value of 25000 mebibytes is recommended. The root file system is automatically resized to fill all available space up to the specified offset. If no value is specified, or if the specified value is smaller than the recommended minimum, the resulting root file system will be too small, and future reinstalls of {op-system} might overwrite the beginning of the data partition.
+
+. Run `openshift-install` again to create Ignition configs from a set of files in the `manifest` and `openshift` subdirectories:
++
+[source,terminal]
+----
+$ openshift-install create ignition-configs --dir $HOME/clusterconfig
+$ ls $HOME/clusterconfig/
+auth  bootstrap.ign  master.ign  metadata.json  worker.ign
+----
+
+Now you can use the Ignition config files as input to the vSphere installation procedures to install {op-system-first} systems.
```

Comment:
Drop the specificity on `vSphere` since this will be used for bare metal installs as well.

```suggestion
Now you can use the Ignition config files as input to the installation procedures to install {op-system-first} systems.
```

Reply:
Oops, misunderstood where this file was being included.  Ignore the suggestions.